.github/workflows/publish-docker-image-to-dockerhub.yaml

```diff
@@ -28,6 +28,7 @@ jobs:
           docker pull $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA
           docker image tag $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA $DOCKERHUB_REPOSITORY/$PACKAGE_NAME:${GITHUB_REF##*/}
           docker image tag $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA $ECR_REGISTRY/$ECR_REPOSITORY:${GITHUB_REF##*/}
+          docker image tag $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA $DOCKERHUB_REPOSITORY/$PACKAGE_NAME:latest
```

Comment:
you added tag to ECR repository but we expected it to be pushed to dockerhub, so it should be added on line 42 i guess...

Reply:
No it's creating a tag `$DOCKERHUB_REPOSITORY/$PACKAGE_NAME:latest` based on `$ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA`

Reply:
hm. shouldn't you push $DOCKERHUB_REPOSITORY/$PACKAGE_NAME:latest in `Publish image to DockerHub` step?

Reply:
There are 2 steps here:
`docker tag Image:Tag OtherImage:OtherTag`

Which is what we do here. 

Down the line, we push the `OtherImage:OtherTag`

Reply:
ok

Reply:
I realized what you meant afterwards. Fixed.